airbyte-integrations/connectors/source-typeform/Dockerfile

```diff
@@ -34,5 +34,5 @@ COPY source_typeform ./source_typeform
 ENV AIRBYTE_ENTRYPOINT "python /airbyte/integration_code/main.py"
 ENTRYPOINT ["python", "/airbyte/integration_code/main.py"]
 
-LABEL io.airbyte.version=1.1.0
```

Comment:
Does that mean this version is re-shipped as both 1.1.0 and 2.0.0? Why'd we decide to ship it as 2.0.0 instead of retroactively making 1.1.0 a breaking change?

Reply:
I.e. if an OSS user tries to upgrade to 1.1.0, I want them to see the modal and have their actors held behind too

Reply:
Yeah I went back and forth on this... If we're using versions as an indication of user expectations and some sort of contract, then I think we need to bump anyway. Though it's still true that 1.1.0 was the one with the breaking change.

What do you think about bumping to 2.0.0, but marking 1.1.0 as the breaking change in the releases? Is that too weird?

Reply:
I think the effect on cloud users going from 1.0.0 -> 2.0.0 would be the same in terms of messaging, but it may be weird to see in migration guides and other places 1.1.0

Reply:
I think I like either 
- 1.1.0 is marked as breaking, don't ship 2.0.0 at all 
- 1.1.0 is marked as breaking, 2.0.0 is shipped with same code (not marked as breaking though) 

then no matter if you upgrade to 1.1.0 or 2.0.0, you end up "upgrading through a breaking change"

Reply:
>  but it may be weird to see in migration guides and other places 1.1.0

I think this is potentially weird, but something we expected to happen sometimes, and I think its the right thing to show

Reply:
Did this one:
> 1.1.0 is marked as breaking, 2.0.0 is shipped with same code (not marked as breaking though)

Reply:
Let's try it this way for now and we can see if it becomes confusing 😕